tests/data/state_estimation/current-sensor/basic-local-current-sensor/sym_output.json

```diff
@@ -0,0 +1,75 @@
+{
+  "version": "1.0",
+  "type": "sym_output",
+  "is_batch": false,
+  "attributes": {},
+  "data": {
+    "node": [
+      {
+        "id": 1,
+        "energized": 1,
+        "u_pu": 0.9999918620376325,
+        "u": 9999.918620376324,
+        "u_angle": 1.222133428418590e-13
+      },
+      {
+        "id": 2,
+        "energized": 1,
+        "u_pu": 0.9999926200230093,
+        "u": 9999.926200230093,
+        "u_angle": 1.204079620603813e-8
+      }
+    ],
+    "line": [
+      {
+        "id": 3,
+        "energized": 1,
+        "p_from": -165467.7877903657,
+        "q_from": -51216.59815742757,
+        "i_from": 10.00053616082199,
+        "s_from": 173212.957722732,
+        "p_to": 165467.9125972454,
+        "q_to": 51185.22352469098,
+        "i_to": 10.0000000005913,
+        "s_to": 173203.8025176185
+      }
+    ],
+    "source": [
+      {
+        "id": 4,
+        "energized": 1,
+        "p": -165467.7877903657,
+        "q": -51216.59815742757,
+        "i": 10.00053616082199,
+        "s": 173212.957722732,
+        "pf": -0.9552852740684438
+      }
+    ],
+    "sym_load": [
+      {
+        "id": 5,
+        "energized": 1,
+        "p": -165467.9125972454,
+        "q": -51185.22352469098,
+        "i": 10.0000000005913,
+        "s": 173203.8025176185,
+        "pf": -0.9553364891074705
+      }
+    ],
+    "sym_voltage_sensor": [
+      {
+        "id": 6,
+        "u_residual": 0.08137962367538165,
+        "u_angle_residual": -1.22213342841859e-13
+      }
+    ],
+    "sym_current_sensor": [
+      {
+        "id": 7,
+        "energized": 1,
+        "i_residual": -5.91296603842798e-10,
+        "i_angle_residual": -6.1368410353424e-11
```

Comment:
why not just make it `0` by means of `rtol` and `atol`? or is this the exact solution?